.github/workflows/submit-learning-path-pr.yml

```diff
@@ -0,0 +1,117 @@
+name: 'Submit Learning Path PR'
+run-name: '[${{ github.ref_name }}] Submit Learning Path PR'
+
+on:
+  workflow_run:
+    workflows: ["Check Learning Path Links"]
+    types:
+      - completed
+
+permissions: {}
+
+jobs:
+  submit-learning-path-pr:
+    name: 'Submit Learning Path PR'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
+        id: check-user
+        with:
+          script: |
+            await github.rest.repos.checkCollaborator({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              username: context.payload.workflow_run.triggering_actor.login
+            });
+    
+      - name: Checkout main
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          persist-credentials: true # We need to persist credentials to push the resulting changes upstream.
+          ref: main
+
+      - name: Get current SHA
+        id: get_new_sha
+        run: |
+          new_sha=$(git rev-parse HEAD)
+          echo "new_sha=$new_sha" >> $GITHUB_ENV
+
+      - name: Get previous update SHA
+        id: get_prev_sha
+        run: |
+          cd .github
+          prev_sha=$(cat learning-path-sha.txt)
+          echo "prev_sha=$prev_sha" >> $GITHUB_ENV
+          cd ../
+
+      # Download the artifact from the workflow that kicked off this one.
+      # The default artifact download action doesn't support cross-workflow
+      # artifacts, so use a 3rd party one.
+      - name: 'Download linting results'
+        uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d
+        with:
+          workflow: ${{env.workflow_name}}
+          run_id: ${{github.event.workflow_run.id }}
+          name: learning-path-review
+          path: ./learning-path-review
+
+      - name: Put Suggestions In Env Var
+        run: |
+          suggestions=$(cat ./learning-path-review/suggestions)
+          echo "suggestions=$suggestions" >> $GITHUB_ENV
+
+      - name: Update Learning Path Hashes
+        id: update-hashes
+        uses: ./.github/actions/apply-learning-path-updates
+        with:
+          learningPathsDirectory: 'documentation/learningPath'
+          learningPathHashFile: '.github/learning-path-sha.txt'
+          suggestions: ${{ env.suggestions }}
+          oldHash: ${{ env.prev_sha }}
+          newHash: ${{ env.new_sha }}
+
+      - name: Create PR Body
+        run: |
+          manuallyReview=$(cat ./learning-path-review/manuallyReview)
+          manuallyReview=${manuallyReview//,/<br />}
+          manuallyReviewSection="<h2>Manually Review:</h2>${manuallyReview}<br />"
+          if test -z "$manuallyReview"; then
```

Comment:
Let's switch all of these if statements to use the bracket commands instead of explicit `test` as that's more conventional in bash scripts.

e.g. `if [ -z "$manuallyReview" ]`